docs/guide/basket/basket_templates.md

```diff
@@ -0,0 +1,83 @@
+# Basket templates
+
+## Template list
+
+|Path|Description|
+|--- |--- |
+|`SilversolutionsEshopBundle:Basket:show.html.twig`|Main basket template|
+|`SilversolutionsEshopBundle:Basket:messages.html.twig`|Renders a message when the basket is modified|
+|`SilversolutionsEshopBundle:Fieldtypes:StockField.html.twig`|Renders the stock Field representing product availability|
+|`SilversolutionsEshopBundle:parts:stock_legend.html.twig`|Renders the legend of symbols used to indicate product availability|
+|`ezcommerce-base-design/Resources/views/themes/standard/basket/widget.html.twig`|Renders the basket preview in the upper right corner of the shop screen|
+|`ezcommerce-base-design/Resources/views/themes/standard/basket/row.html.twig`|Renders one row of basket preview|
+|`SilversolutionsEshopBundle:Basket:stored_basket_preview_wish_list.html.twig`|Renders the wishlist row with  the number of products in the "My Shop" menu|
+|`SilversolutionsEshopBundle:Basket:stored_basket_preview_comparison.html.twig`|Renders the comparison row with the number of products in the "My Shop" menu|
+
+## Display the content of a basket
+
+The `BasketController` provides the [`basket`](basket_api/basket_data_model.md) variable containing the content of the current basket.
+
+You can access all basket attributes from a template.
+
+The count of basket lines:
+
+``` html+twig
+{{ basket.lines.count }}
+```
+
+Access to basket lines:
+
+``` html+twig
+{% for line in basket.lines %}
+    {{ line.sku }} 
+    {{ line.linePriceAmountGross }} 
+{% endfor %}
+```
+
+You can access the basket total value (net or gross) using `basket.totalsSum`:
+
+``` html+twig
+{{ basket.totalsSum.totalGross }}
+```
+
+To get the catalog element from a basket line, use `line.catalogElement`.
```

Comment:
```suggestion
To get a catalog element from the basket line, use `line.catalogElement`.
```

Reply:
There is only one catalog element per line, so it should be:
```suggestion
To get the catalog element from the basket line, use `line.catalogElement`.
```